### DISCUSSION TITLE
White Markers?

### DISCUSSION THREAD

**Discussion by GutHib** [organicmaps/organicmaps#4876] [Ideas]
Hi - in my view, it would be great to have simple white markers along with the colorful ones. For my purpose, I travel around the world and use the markers to remember the interesting places/bars/restaurants I might revisit years later. The white markers would be useful to mark the places I've been to, but that are NOT worth visiting again.

**Comment by j13m126** [comment]
You could try using bookmark lists. One for places you want to visit again and one for the places you don't want to visit again.

**Reply by GutHib** [reply]
My maps are splattered with color-coded markers - red means something, brown means something, orange means something... I'm assuming that there are no white markers because they are hard to see, but for people like me, that would be a feature, because I wouldn't want those places to stick out. Bookmark lists wouldn't do the trick in any way.

**Comment by Misalf-git** [comment]
I guess when SVG icons become a thing (assuming marker icons are currently PNG), it would be easy to allow the user to choose any color (+ alpha), in addition to the current predefined colors.

**Reply by dcht00** [reply]
Is there any progress on this? It would be great to have markers of *any* color. Adding extra shapes as well, so SVG markers seem like a great idea too.

**Reply by biodranik** [reply]
There are several related issues and even a started pull request. You may search and link related issues here. It requires significant time and effort to refactor, so the progress is slow.
